tensorboard/webapp/widgets/line_chart_v2/lib/worker.ts

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+/**
+ * Returns a Worker instancefor a given resource url.
```

Comment:
Maybe “Creates a new worker that loads and executes the JavaScript at
the given URL”? to clarify the semantics of the resource path and also
that this is creating something rather than “getting” it from a cache.